### PR TITLE
Kepler 33

### PR DESCRIPTION
**Pull request by seityaya**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/kepler_33/readme.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Kepler-33
```
Comment (drashna) [inline keyboards/kepler_33/readme.md:1]: Please use the default template:
```c
# %KEYBOARD%

![%KEYBOARD%](imgur.com image replace me!)

*A short description of the keyboard/project*

* Keyboard Maintainer: [%REAL_NAME%](https://github.com/%USER_NAME%)
* Hardware Supported: *The PCBs, controllers supported*
* Hardware Availability: *Links to where you can find this hardware*

Make example for this keyboard (after setting up your build environment):

    make %KEYBOARD%:default

Flashing example for this keyboard:

    make %KEYBOARD%:default:flash

See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).

## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `QK_BOOT` if it is available
```

File: keyboards/kepler_33/readme.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+[![Foo](kepler-33.png)](https://www.keyboard-layout-editor.com/#/gists/a456c2256174a2cfa0b4aea832e78d8e)
```
Comment (drashna) [inline keyboards/kepler_33/readme.md:3]: Please rehost image on Imgur or similar dedicate image hosting site.

File: keyboards/kepler_33/readme.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# Kepler-33
+
+![Kepler-33](https://i.imgur.com/zVvzBrO.png)
```
Comment (drashna) [inline keyboards/kepler_33/readme.md:3]: This uses a smaller, but still high res version of the image, that is more suitable for github markdown rendering. 
```suggestion
![Kepler-33](https://i.imgur.com/zVvzBrOl.png)
```

**Review by drashna — APPROVED** [review]

File: keyboards/kepler_33/readme.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
+# Kepler-33
+
+![Kepler-33](https://i.imgur.com/zVvzBrOl.png)
+
+*True 40% keyboard. The simple and symmetric.*
+
+* Keyboard Maintainer: [Yaya](https://github.com/seityaya)
+* Hardware Supported: *Kepler-33/proto, STM32F401*
```
Comment (fauxpark) [inline keyboards/kepler_33/readme.md:8]:
```suggestion
True 40% keyboard. The simple and symmetric.

* Keyboard Maintainer: [Yaya](https://github.com/seityaya)
* Hardware Supported: Kepler-33/proto, STM32F401
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**